### PR TITLE
gas and readme

### PR DESCRIPTION
**Pull request by shogochiai**
## What
- Fee estimation env val
- add aggregator command to the readme
- block explorer URL update

### CODE REVIEW

File: packages/wallet/src/helpers/ContractManager.ts [hunk 1]
```diff
@@ -28,7 +31,7 @@ export class ContractManager {
       operationKindType
     )
 
-    return result[0].high + 40000
+    return result[0][FEE_GRADE ? FEE_GRADE : "high"] + (FEE_BONUS ? FEE_BONUS : 0);
```
Comment (daiki44) [inline packages/wallet/src/helpers/ContractManager.ts:34]: Could you use lint?

File: packages/cli/README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ node script/addr2hex.js KT1....XXX # When you deployed a new contract.
 cp .sample.env .env
 vi .env # Replace: CONTRACT_BASE58,{DEPOSIT,COMMITMENT,ADJUDICATION}_CONTRACT_ADDERSS
 lerna clean -y && lerna bootstrap && lerna run build
+node ../plasma-aggregator/lib/index.js & >/dev/null 2>&1
```
Comment (daiki44) [inline packages/cli/README.md:12]: I think it is easier to understand the aggregator in the foreground when testing it.

File: packages/wallet/src/helpers/ContractManager.ts [hunk 3]
```diff
@@ -8,10 +8,13 @@ import {
   TezosMessageUtils
 } from 'conseiljs'
 import { TzWallet } from '../'
+const FEE_GRADE = process.env.FEE_GRADE as string
```
Comment (daiki44) [inline packages/wallet/src/helpers/ContractManager.ts:11]: Is it possible to use .env that we wrote in the cli package here?

File: packages/contract/src/DepositContract.ts [hunk 4]
```diff
@@ -100,7 +100,7 @@ export class DepositContract implements IDepositContract {
       JSON.stringify(param)
     )
     console.log(
-      `succeed to deposit. open https://arronax.io/tezos/carthagenet/operations/${result.operationGroupID.replace(
+      `succeed to deposit. open https://arronax.io/tezos/carthagenet/operation_group/${result.operationGroupID.replace(
```
[the remark this answers is not in the document]
Comment (shogochiai) [reply packages/contract/src/DepositContract.ts]: Sorry it was operation_groups. I've fixed it.

You'll get a modal detail view by https://arronax.io/tezos/carthagenet/operation_groups/ooRXxkX7mmi8JQ6Q1MwaFXNC8jMHu2eQ1KoDtkFdcLK2wXqmGfD

But you won't by https://arronax.io/tezos/carthagenet/operations/ooRXxkX7mmi8JQ6Q1MwaFXNC8jMHu2eQ1KoDtkFdcLK2wXqmGfD

File: packages/wallet/src/helpers/ContractManager.ts (hunk 3 above)
Comment (shogochiai) [reply packages/wallet/src/helpers/ContractManager.ts]: Yes because the lerna module in in the node_modules and it is imported in the index.ts

**Pull request merged**